test/Twig/Tests/Profiler/Dumper/AbstractTest.php

```diff
@@ -0,0 +1,46 @@
+<?php
+
+/*
+ * This file is part of Twig.
+ *
+ * (c) Fabien Potencier
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+abstract class Twig_Tests_Profiler_Dumper_AbstractTest extends PHPUnit_Framework_TestCase
+{
+    abstract public function testDump();
```

Comment:
I suggest removing this abstract method. It is not really needed given that it is not used in the abstract class